projects/app/sidepanel.css

```
@@ -34,8 +34,20 @@
 
 body {
   font-family:
-    -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu,
-    Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+    "Roboto",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)